quest/wavefunction.py

```diff
@@ -0,0 +1,42 @@
+"""
+A basic wave function class
+"""
+
+import psi4
+
+
+class WaveFunction(object):
+    """
+    Basic wave function class
+
+    Members:
+        options: dictionary with various options, including the basis name.
+        mints: contains a psi4 mints object built from the basis set.
+        energies: a dictionary of various energies that have been
+                  calculated (including scf, nuclear, mp2, etc.).
+        arrays: a dictionary of various arrays (including a coefficient
+                matrix, a fock matrix, etc.)
+    """
+
+    def __init__(self, mol):
+        """
+        Initialize the WaveFunction class.
+
+        Args:
+            mol: a Molecule class from the QuESt repository. Needs to
+                 have a basis set at least.
+        """
+
+        # Set whichever options you would like. The only default is the
+        # basis set name.
+        self.options = {'basis_name': mol.bas.name}
```

Comment:
Okay. I can add it, but they will need to add a "basisset()" function in their molecule class for that to work:

https://github.com/MolSSI-SSS/SSS_2017_QuESt/blob/master/quest/molecule.py